src/main/java/org/alliancegenome/curation_api/dao/VocabularyDAO.java

```diff
@@ -11,4 +12,10 @@
     protected VocabularyDAO() {
         super(Vocabulary.class);
     }
+    
+    public Long getIdFromName(String name) {
+        Query jpqlQuery = entityManager.createQuery("SELECT vocabulary.id FROM Vocabulary vocabulary WHERE vocabulary.name=:name");
+        jpqlQuery.setParameter("name", name);
+        return(Long) jpqlQuery.getSingleResult();
+    }
```

Comment:
There is already a "find" framework in place so that we don't need this extra code, its similar to the "search" methods.